scaletest/templates/scaletest-runner/scripts/run.sh

```diff
@@ -6,54 +6,61 @@ set -euo pipefail
 # shellcheck disable=SC2153 source=scaletest/templates/scaletest-runner/scripts/lib.sh
 . "${SCRIPTS_DIR}/lib.sh"
 
+mapfile -t scaletest_load_scenarios < <(jq -r '. | join ("\n")' <<<"${SCALETEST_PARAM_LOAD_SCENARIOS}")
+export SCALETEST_PARAM_LOAD_SCENARIOS=("${scaletest_load_scenarios[@]}")
+
 log "Running scaletest..."
 set_status Running
 
 start_phase "Creating workspaces"
 coder exp scaletest create-workspaces \
-	--count "${SCALETEST_NUM_WORKSPACES}" \
-	--template "${SCALETEST_TEMPLATE}" \
-	--concurrency "${SCALETEST_CREATE_CONCURRENCY}" \
-	--job-timeout 15m \
+	--count "${SCALETEST_PARAM_NUM_WORKSPACES}" \
+	--template "${SCALETEST_PARAM_TEMPLATE}" \
+	--concurrency "${SCALETEST_PARAM_CREATE_CONCURRENCY}" \
+	--job-timeout 2h \
 	--no-cleanup \
 	--output json:"${SCALETEST_RESULTS_DIR}/create-workspaces.json"
 show_json "${SCALETEST_RESULTS_DIR}/create-workspaces.json"
 end_phase
 
-wait_baseline 5
-
-start_phase "SSH traffic"
-coder exp scaletest workspace-traffic \
-	--ssh \
-	--bytes-per-tick 10240 \
-	--tick-interval 1s \
-	--timeout 5m \
-	--output json:"${SCALETEST_RESULTS_DIR}/traffic-ssh.json"
-show_json "${SCALETEST_RESULTS_DIR}/traffic-ssh.json"
-end_phase
-
-wait_baseline 5
+wait_baseline "${SCALETEST_PARAM_LOAD_SCENARIO_BASELINE_DURATION}"
 
-start_phase "ReconnectingPTY traffic"
-coder exp scaletest workspace-traffic \
-	--bytes-per-tick 10240 \
-	--tick-interval 1s \
-	--timeout 5m \
-	--output json:"${SCALETEST_RESULTS_DIR}/traffic-reconnectingpty.json"
-show_json "${SCALETEST_RESULTS_DIR}/traffic-reconnectingpty.json"
-end_phase
-
-wait_baseline 5
-
-start_phase "Dashboard traffic"
-coder exp scaletest dashboard \
-	--count "${SCALETEST_NUM_WORKSPACES}" \
-	--job-timeout 5m \
-	--output json:"${SCALETEST_RESULTS_DIR}/traffic-dashboard.json"
-show_json "${SCALETEST_RESULTS_DIR}/traffic-dashboard.json"
-end_phase
+for scenario in "${SCALETEST_PARAM_LOAD_SCENARIOS[@]}"; do
+	start_phase "Load scenario: ${scenario}"
+	case "${scenario}" in
+	"SSH Traffic")
+		coder exp scaletest workspace-traffic \
+			--ssh \
+			--bytes-per-tick 1024 \
+			--tick-interval 100ms \
```

Comment:
suggestion: we can also expose these as parameters to allow heavier load generation